lib/main.dart

```diff
@@ -17,14 +18,16 @@ class FlameGame extends Game with KeyboardEvents, TapDetector {
   late Ball ball;
   double ballDirection = 1;
 
+  bool isCatch = false;
+
   late double halfGameWidth;
 
   double direction = 0;
   double positionX = 500;
-  Vector2 ballPosition = Vector2(500, 500);
+  Vector2 ballPosition = Vector2(500, 600);
 
   double playerSpeed = 500;
```

Comment:
I think we should to talk in percent of screen not in pixel directly think responsive 😉

Reply:
Ok why not Mister english